crates/astria-sequencer/src/proposal/commitment.rs

```diff
@@ -26,18 +26,32 @@ use crate::transaction::Signed;
 pub(crate) fn generate_sequence_actions_commitment(
     txs_bytes: Vec<Bytes>,
 ) -> ([u8; 32], Vec<Bytes>) {
-    // ignore any transactions that are not deserializable
+    use proto::{
+        generated::sequencer::v1alpha1 as raw,
+        Message as _,
+    };
     let txs = txs_bytes
         .into_iter()
-        .filter_map(|tx_bytes| match Signed::try_from_slice(&tx_bytes) {
-            Ok(tx) => Some((tx, tx_bytes)),
-            Err(err) => {
-                tracing::debug!(tx_bytes = hex::encode(tx_bytes), error = ?err, "failed to deserialize tx bytes");
-                None
-            }
+        .filter_map(|bytes| {
+            raw::SignedTransaction::decode(&*bytes)
+            .map_err(|err| {
+                info!(error = ?err, "failed to deserialize bytes as a signed transaction");
```

Comment:
info seems high potentially

Reply:
Do you mean it should be `warn!` or `debug!`?

Reply:
probably `debug!` here

Reply:
Changed it to debug